src/toniarts/openkeeper/animation/PoseTrack.java

```diff
@@ -57,9 +52,9 @@
 /**
  * A single track of pose animation associated with a certain mesh.
  */
-public final class PoseTrack implements Track, ClonableTrack {
+public final class PoseTrack extends MorphTrack {
 
-    private int targetMeshIndex;
+    private Geometry target;
     private PoseTrack.PoseFrame[] frames;
     private float[] times;

```

Comment:
I guess this is now redundant with
https://github.com/jMonkeyEngine/jmonkeyengine/blob/84d7d3bb2ddc434a65ef06ac789c245d21e1e359/jme3-core/src/main/java/com/jme3/anim/MorphTrack.java#L56